### DISCUSSION TITLE
problème de rendu de devoir

### DISCUSSION THREAD

**Discussion by Anicq** [fortierq/oc-m1-2021#17] [General]
Bonjour Monsieur,
En essayant de rendre mon devoir, je n'ai pas trouvé ce que nous avez indiqué de chercher ("edit this file"), à la place j'ai créé un nouveau dossier sur lequel j'ai copié mon code mais maintenant je n'arrive plus à le retrouver. 
Pourriez vous me dire si vous le voyez ou non?
Aurore NICQ MPSI

**Comment by fortierq** [comment]
Bonjour,

Non je ne le vois pas.
Mais vous pouvez aussi me l'envoyer par mail finalement : qpfortier@gmail.com
